problemsets-mat223/linearalbegra.tex

```diff
@@ -3084,10 +3084,57 @@ \section*{Task 2.3: Pat and Jamie}
 	and let $\mathcal R$ be rotation counter-clockwise by $90^\circ$.
 	\begin{parts}
 		\item Find a matrix $P$ so that $P\vec x=\mathcal P(\vec x)$ for all $\vec x\in\R^2$.
+			\begin{solution}
+				$P = \frac{1}{13}\mat{4&6\\6&9}$ is such a matrix.
+
+				We know from an earlier exercise that 
+				$\mathcal P(\vec x)=\dfrac{\vec x\cdot\vec u}{\norm{u}^2}\vec u$.
+				So if $\Set{\vec e_1,\vec e_2}$ is the standard basis for $\R^2$,
+				we can compute
```

Comment:
I'd like you to do these by plugging in e_1 and e_2 (which are `\xhat` and `\yhat` in this document) and then pointing out that these give the first and second columns

Reply:
I can do it that way.

I see that you define `\xhat` and `\yhat` in the preamble, but they never appear in the rest of the document, and you call the standard basis vectors \vec e_1 and \vec e_2 every time.

Is this something you want me to change everywhere?

Reply:
Yes, please. Unless we talk about an arbitrary e_i, we should use \xhat, etc. This way someone could recompile the document for their physics class :-)

Reply:
Fair enough. I agree that it would be nicer that way. 

Is it okay if I save the changing e_1,e_2 into \xhat,\yhat throughout the whole document to another commit after I've finished all the problems? I can change them as I come across them, but I'd rather wait until I'm done the later problems before going back to the beginning and changing the stuff I've gone over already.